src/sparsezoo/objects/directories.py

```diff
@@ -280,16 +280,17 @@ class OnnxGz(Directory):
     """
     Special class to handle onnx.model.tar.gz files.
     Desired behavior is that all information about files included in the tarball are
-    available however, when the file.path is accessed, it will point only to the
-    `model.onnx` as this is the expected behavior for loading, additionally,
+    available however, when the file's `path` property is accessed, it will point only
```

Comment:
annoying, but apostrophes `'` are not well supported by Sphinx, they tend to parse these as unclosed strings